### PR TITLE
Replace "default constructor" with "parameterless constructor"

### PR DESCRIPTION
**Pull request by Youssef1313**
Contributes to: #11957

I've also made some minor changes in these files.

### CODE REVIEW

**Review by cartermp — COMMENTED** [review]
Generally I'm in favor, though I can't speak towards the article under `/framework` since I don't own that area.

File: docs/framework/xaml-services/markup-extensions-for-xaml-overview.md [hunk 1]
```diff
@@ -7,147 +7,164 @@ helpviewer_keywords:
...
+
+When a markup extension usage is accessed, the XAML object writer can provide services to a custom <xref:System.Windows.Markup.MarkupExtension> class through a service connection point in the <xref:System.Windows.Markup.MarkupExtension.ProvideValue%2A?displayProperty=nameWithType> override. The services can be used to obtain context about the usage, specific capabilities of the object writer, XAML schema context, and so on.
+
+<a name="XAML_Defined_Markup_Extensions"></a>
+## XAML-Defined markup extensions
```
Comment (rpetrusha) [inline docs/framework/xaml-services/markup-extensions-for-xaml-overview.md]:
```suggestion
## XAML-defined markup extensions
```

File: docs/fsharp/language-reference/attributes.md [hunk 2]
```diff
@@ -19,13 +19,13 @@ In the previous syntax, the *target* is optional and, if present, specifies the
...
+The *arguments* are the arguments to the constructor for the attribute type. If an attribute has a parameterless constructor, the argument list and parentheses can be omitted. Attributes support both positional arguments and named arguments. *Positional arguments* are arguments that are used in the order in which they appear. Named arguments can be used if the attribute has public properties. You can set these by using the following syntax in the argument list.
 
 ```fsharp
 property-name = property-value
 ```
 
-Such property initializations can be in any order, but they must follow any positional arguments. Following is an example of an attribute that uses positional arguments and property initializations.
+Such property initializations can be in any order, but they must follow any positional arguments. Following is an example of an attribute that uses positional arguments and property initializations:
```
Comment (rpetrusha) [inline docs/fsharp/language-reference/attributes.md]:
```suggestion
Such property initializations can be in any order, but they must follow any positional arguments. The following is an example of an attribute that uses positional arguments and property initializations:
```

**Review by rpetrusha — APPROVED** [review]
Your changes look good, @Youssef1313. I've left two minor suggestions.

**Pull request merged**